src/modules/navigation/components/Navigation/Navigation.consts.tsx

```diff
@@ -90,7 +90,14 @@ export const getNavigationItemsLeft = (
 			}
 		),
 		id: 'visitor-spaces',
-		active: currentPath === ROUTES.home,
+		active:
+			currentPath === ROUTES.home ||
+			Object.keys(ROUTE_PREFIXES).every(
+				(key) =>
+					!currentPath.startsWith(
+						`/${(ROUTE_PREFIXES as { [key: string]: string })[key]}`
+					)
```

Comment:
this works for now, but will not work once we add content pages since those can have any url.
We'll have to make this an async check, so that we can query the backend if this space exists, if it does, then this should be the active navigation item.

Let's merge as is, then fix it once we add content pages.

Reply:
https://meemoo.atlassian.net/browse/ARC-790